lib/ui/views/forum/forum-post/forum_post_viewmodel.dart

```diff
@@ -156,7 +156,7 @@ class PostViewModel extends BaseViewModel {
   ) {
     List urlPart = url!.split('{size}');
     String avatarUrl = '';
-    String baseUrl = 'https://forum.freecodecamp.org';
+    String baseUrl = 'https://mobilefccinstance.com';
```

Comment:
Shouldn't this url come from the `getCurrentUrl` function you defined?

Reply:
Yes this line can be removed